### ISSUE TITLE
php - composer question

### ISSUE THREAD

**Issue by dpreilan** [ccxt/ccxt#862]
I now there has been recent work for php. I am on version 1.10.389 (from ccxt.php)

Does anybody have a working example of a php testcase with composer. 

I have a simple program:
```JSON
composer.json:
{
    "require": {
        "curl/curl": "^1.8",
        "lib-pcre": "^8.32",
        "lib-iconv": "^2.17",
        "ccxt/ccxt": "^1.10"
    }
}
```
foo.php
```PHP
<?php

require_once __DIR__ . '/vendor/autoload.php';

use ccxt;

var_dump (\cxxt\Exchange::$exchanges);
?>
```
php -f foo.php fails with:
```
PHP Warning:  The use statement with non-compound name 'ccxt' has no effect in /home/livenano/bitcoin/php/foo.php on line 5
PHP Fatal error:  Class 'cxxt\Exchange' not found in /home/livenano/bitcoin/php/foo.php on line 8
```

What am I missing?

**Comment by kroitor** [comment]
1. Delete the `use ccxt;` line
2. Fix the typo in the last line: `var_dump (\cxxt\Exchange::$exchanges);` → `var_dump (\ccxt\Exchange::$exchanges);` (cxxt → ccxt)
3. Retry

**Comment by dpreilan** [comment]
I must have messed that up attempting to rootcause. Now,

PHP Fatal error:  Class 'ccxt\Exchange' not found in /home/livenano/bitcoin/php/foo.php on line 6

**Comment by dpreilan** [comment]
Note, the following works if when I add the second require
<?php

require_once __DIR__ . '/vendor/autoload.php';

require('vendor/ccxt/ccxt/ccxt.php');

//var_dump (\cxxt\Exchange::$exchanges);
var_dump (\ccxt\Exchange::$exchanges);

?>

**Comment by kroitor** [comment]
This is fixed in #876, ccxt version 1.10.412. Let us know if you have any difficulties with it. Thx!
